web/src/components/onboarding/OnboardingNexusDbaName.tsx

```diff
@@ -0,0 +1,21 @@
+import { OnboardingField, OnboardingProps } from "@/components/onboarding/OnboardingField";
+import { ProfileFieldErrorMap } from "@/lib/types/types";
+import { ReactElement } from "react";
+
+interface Props extends Omit<OnboardingProps, "fieldName"> {
+  fieldStates?: Partial<ProfileFieldErrorMap>;
+}
+
+export const OnboardingNexusDbaName = (props: Props): ReactElement => {
```

Comment:
doesn't this already exist? `ProfileNexusDBANameField`?

Reply:
We shouldn't have two components for the same field, they will need to be combined somehow